tests/programming_languages/prog_lang.js

```diff
@@ -0,0 +1,24 @@
+var file_path = process.env.FILE_PATH
+var webdriver = require('selenium-webdriver');
+var assert = require('chai').assert;
+var { Builder, By } = require("selenium-webdriver");
+
+describe("Testing Navigation on HtmlForms", function() {
+    this.timeout(3 * 1000 * 60);
+    var driver = new webdriver.Builder().forBrowser('chrome').build()
+
+    before(function setupWebdriver(done) {
+        var file_url = `file://${file_path}`;
+        driver.get(file_url).then(done)
+    })
+    
+    after(function() {
+        driver.quit()
+    })
+
+    it('test case: assert that the url to the document is present', async () => {
+        const body = await driver.findElements(By.css('body'));
```

Comment:
The file that they are submitting is in `.txt` file. This is reading an html file